src/sdk-reference/kuzzle/add-listener/add-listener.js.md

```diff
@@ -0,0 +1,23 @@
+```javascript
+/**
+ * @param {string} event
+ * @param {function} callback
+ * @returns {Kuzzle} this
+ */
+addListener(event, callback)
+```
+
+## Arguments
+
+| Argument | Type | Description | Required |
+|--------|------|-------------|------------ |
+| `event` | string | One of the event described in the `Event Handling` section of this documentation  | yes |
```

Comment:
We should have a link to the Event Handling section, even if the link is dead for now...